superbench/benchmarks/micro_benchmarks/build.sh

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation - All rights reserved
+# Licensed under the MIT License
+
+ #!/bin/bash
+
+SB_PATH="${SB_PATH:-/usr/local/bin}"
+SB_LIB="${SB_LIB:-/usr/local/lib}"
+
+for dir in */ ; do
+    if [ -f $dir/CMakeLists.txt ]; then
```

Comment:
why not write cmake file for this? e.g.,
```cmake
SB_PATH ?= /usr/local/bin

FILE(GLOB subdirs /*)
FOREACH(subdir ${subdirs})
  # cmake for each subdir ...
  ADD_SUBDIRECTORY(${subdir})
ENDFOREACH()
```

Reply:
Is there any necessary factors that to use cmake for build script? I think shell script is easy for caller to use, and it has more flexibility in the future.